Homework6.py

```diff
@@ -0,0 +1,157 @@
+import matplotlib.pyplot as plt
+import random
+plt.style.use('ggplot')
+
+''' Our aim was to create an improved model
+    of bacterial cells growth and death in vitro,
+    taking into consideration inappropriate environment
+    The original model may be checked at the 14 page of
+    "BRS Microbiology and Immunology, 6th edition"
+    This script contains:
+    class Bacteria - to describe bacterial cells
+    class Medium - to describe environment and the experiment
+    class Scientist - to describe and visualize the results
+    and to describe the scientist
+    function go_on - to start and run the experiment properly'''
+
+
+class Bacteria(object):
+
+    exp_data = list()
```

Comment:
it's not good idea to keep all bacterias list and count in class variable for bacteria. 
we should create bacterias class for this.
overall, class variables are rarely used option. Here it's not needed.